changelog/1.18.0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "1.18.0"
+description: "Released on 04/21/2021"
+---
+
+### Breaking changes ❗
+
+- web: Renamed environments to workspaces
+
+### Features ✨
+
+- web: Moved _built-in_ sign-in behind a toggle so that it does not display when
+  OIDC is configured for authentication
+- web: Added dev URL support in workspaces as code templates
+- web: Added admin toggle for workspaces as code feature
+- web: Added Bitbucket Server support for workspaces as code
+- infra: Added `CODER_WP_NAME` environment variable to workspaces
+- web: Updated color scheme on sign-in page
+- web: Improved error notifications with additional details and resolutions.
+  Added _Tip_, _Error Type_, and enumerations for workspaces as code errors.
+- infra: Added a background job to update templates
+- web: Added a workspace status indicator in the **Switch Workspace** selector
+
+### Bug fixes 🐛
+
+- web: Fixed bug causing re-authentication whenever an external link to Coder is
+  used. Most notably, re-authentication will not be required when clicking
+  **Open in Coder** buttons
+- web: Fixed the inability to delete an empty organization while workspaces were
+  awaiting deletion in the background
+- web: Improve error messages regarding attempts to import a workspaces as code
+  template from a repository using an SSH URI
+- infra: Prevent updates to environments created via local template
```

Comment:
We'll want to reword this one, just want to get a review from the WAC team a bit for verification